x-pack/test/functional/apps/ml/anomaly_detection/anomaly_explorer.ts

```diff
@@ -51,9 +52,15 @@ const testDataList = [
   },
 ];
 
+const cellSize = 15;
+
+const overallSwimLaneTestSubj = 'mlAnomalyExplorerSwimlaneOverall';
+const viewBySwimLaneTestSubj = 'mlAnomalyExplorerSwimlaneViewBy';
```

Comment:
I did it on purpose because the swim lane visualization is reusable. You can find it as
- embeddable in the dashboard 
- embeddable in the Logs UI 
- it's going to be inside of Lens vis as well